cartographer_rviz/cartographer_rviz/submaps_display.cc

```diff
@@ -54,6 +56,12 @@ SubmapsDisplay::SubmapsDisplay() : tf_listener_(tf_buffer_) {
       "Tracking frame", kDefaultTrackingFrame,
       "Tracking frame, used for fading out submaps.", this);
   client_ = update_nh_.serviceClient<::cartographer_ros_msgs::SubmapQuery>("");
+  submaps_category_ = new ::rviz::Property(
+      "Submaps", QVariant(), "The list of all submaps.", this);
```

Comment:
For consistency, please drop "The" from help text like this. This becomes, "List of all submaps."